src/pages/Admin/Details/Form/DetailsForm.tsx

```diff
@@ -61,7 +60,8 @@ const HANDLED_ERRORS: string[] = [
 
 const DetailsForm: React.FC<DetailsFormProps> = ({ venue, worldId }) => {
   const history = useHistory();
-  const venueId = useVenueId();
+  const venueId = venue?.id;
```

Comment:
[NOTE]
Interesting how `worldId` is being passed down here since `venue` might be undefined. But the URL `spaceSlug` wouldn't (just like the old `venueId`).

The `venueId` maybe can be taken from the URL as to not possibly introduce some regression. This is used in many other places:
```tsx
  const spaceSlug = useSpaceParams();
```

Reply:
> but the correct checks have been made in the parent component

Good enough

Reply:
If the parent is also always making sure `venue` exists, maybe that `?.` is unnecessary